<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>VideoSettings</class>
 <widget class="QWidget" name="VideoSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>624</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Video Settings</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="enableVideoGroupBox" >
        <property name="title" >
         <string>Enable video</string>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QGroupBox" name="webcamSelectionGroupBox" >
           <property name="title" >
            <string>Webcam Selection</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Webcam device:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0" >
               <widget class="QPushButton" name="webcamPreviewButton" >
                <property name="text" >
                 <string>Webcam Preview</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QComboBox" name="webcamDeviceComboBox" />
              </item>
             </layout>
            </item>
            <item row="0" column="1" >
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="0" column="0" >
               <widget class="QLabel" name="webcamPreviewLabel" >
                <property name="minimumSize" >
                 <size>
                  <width>128</width>
                  <height>128</height>
                 </size>
                </property>
                <property name="maximumSize" >
                 <size>
                  <width>128</width>
                  <height>128</height>
                 </size>
                </property>
                <property name="pixmap" >
                 <pixmap resource="../wengophone.qrc" >:/pics/iconbar/webcam.png</pixmap>
                </property>
                <property name="alignment" >
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Connection Type</string>
           </property>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="0" >
             <widget class="QTreeWidget" name="videoQualityTreeWidget" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>130</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>16777215</width>
                <height>130</height>
               </size>
              </property>
              <property name="rootIsDecorated" >
               <bool>false</bool>
              </property>
              <property name="itemsExpandable" >
               <bool>false</bool>
              </property>
              <column>
               <property name="text" >
                <string>Video quality</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>Down (kbit/s)</string>
               </property>
              </column>
              <column>
               <property name="text" >
                <string>Up (kbit/s)</string>
               </property>
              </column>
              <item>
               <property name="text" >
                <string>Normal</string>
               </property>
               <property name="text" >
                <string>0 to 512</string>
               </property>
               <property name="text" >
                <string>0 to 128</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Good</string>
               </property>
               <property name="text" >
                <string>512 to 2048</string>
               </property>
               <property name="text" >
                <string>128 to 256</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Very good</string>
               </property>
               <property name="text" >
                <string>2048+</string>
               </property>
               <property name="text" >
                <string>256+</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Excellent</string>
               </property>
               <property name="text" >
                <string>8192+</string>
               </property>
               <property name="text" >
                <string>1024+</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Video quality depends on your Internet connection bandwidth</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>&lt;b>Warning!&lt;/b> When used for the first time we recommend that you select&lt;br>the normal video quality mode, then you can try different modes to find out&lt;br>the most suitable one&lt;br>&lt;br>&lt;b>Tips&lt;/b>: Go to http://test.lvcm.com to determine your Internet connection bandwidth</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="makeTestVideoCallButton" >
     <property name="text" >
      <string>Make a test video call</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>151</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>webcamDeviceComboBox</tabstop>
  <tabstop>webcamPreviewButton</tabstop>
  <tabstop>videoQualityTreeWidget</tabstop>
  <tabstop>makeTestVideoCallButton</tabstop>
 </tabstops>
 <resources>
  <include location="../wengophone.qrc" />
 </resources>
 <connections/>
</ui>
